cmd/check_container.go

```diff
@@ -30,6 +32,8 @@ var checkContainerCmd = &cobra.Command{
 		// Expect exactly one positional arg. Check here instead of using builtin Args key
 		// so that we can get a more user-friendly error message
 
+		log.Info("certification library version ", version.Version.String())
```

Comment:
Nit: This could just go at the end of the preRunConfig. That would log it very early, and only need to be in one place.

Reply:
I was thinking that might work better, but then it would show up in the Certify and Support workflows, and I'm not sure we want it to be there.

https://github.com/redhat-openshift-ecosystem/openshift-preflight/blob/main/cmd/certify.go#L13
https://github.com/redhat-openshift-ecosystem/openshift-preflight/blob/main/cmd/support.go#L30